game-core/src/test/java/games/strategy/triplea/delegate/WW2V3Year41Test.java

```diff
@@ -1166,7 +1164,7 @@ void testAttackDestroyerAndSubsAgainstSubAndDestroyer() {
         invocation -> {
           final Collection<Unit> selectFrom = invocation.getArgument(0);
           return new CasualtyDetails(
-              Collections.singletonList(selectFrom.iterator().next()), new ArrayList<>(), false);
+              List.of(selectFrom.iterator().next()), new ArrayList<>(), false);
```

Comment:
List.of() can be used for the next param as well. I checked CasualtyDetails and it makes a copy.